packages/ng/libraries/root/src/style/components/_dropdown.scss

```diff
@@ -0,0 +1,97 @@
+.lu-dropdown {
+	margin: calc(#{_theme("spacings.smaller", true)} * -1);
+	margin: calc(#{_theme("spacings.smaller")} * -1);
+}
+
+.lu-dropdown-options,
+.lu-dropdown-options-item-children {
+	border-radius: _theme("commons.border.radius", true);
+	border-radius: _theme("commons.border.radius");
+	list-style-type: none;
+	padding: 0;
+	margin: 0;
+	width: 14rem; // magic number
```

Comment:
these comments should be removed ?

Reply:
I think it avoids the question of where the value comes from.

Reply:
or, or, or, this should be a theme variable

Reply:
It's only about this component, so I don't know.